addons/base_import/static/src/import_action/import_action.js

```diff
@@ -50,6 +51,28 @@ export class ImportAction extends Component {
             previewError: "",
         });
 
+        this.uploadFiles = useFileUploader();
+        useDropzone(useRef("root"), async event => {
+            const file = Array.from(event.dataTransfer.files).find(file => {
+                return file.name.endsWith(".csv")
```

Comment:
As a wise man would say: "boarf :boar:". Here, I call one function instead of chaining two functions. The function is standard and self-explanatory: It returns true if the string `field.name` ends with `.csv`.